core/tests/protractor/preferences.js

```diff
@@ -19,30 +19,35 @@
 var general = require('../protractor_utils/general.js');
 var users = require('../protractor_utils/users.js');
 var workflow = require('../protractor_utils/workflow.js');
+var PreferencesPage = require('../protractor_utils/PreferencesPage');
```

Comment:
I took a look at the Travis log and I think the most recent round of changes (toBe --> toMatch) should be reverted. If I'm not mistaken, the issue is related to there being no ".js" at the end of this string.

Reply:
okay